lib/constants.js

```diff
@@ -1,5 +1,13 @@
 import toHash from 'tohash'
 
+export const DEFAULT_ASPECT_RATIO = 1
```

Comment:
The default should be how Carbon production currently is, which if I am not mistake is "Auto" in this case